### ISSUE TITLE
NFS utils not pre-installed on CentOS 6.4 VBox

### ISSUE THREAD

**Issue by drock** [puppetlabs/puppet-vagrant-boxes#30]
The centos-64-x64-vbox4210.box base box does not by default have the nfs-utils and  nfs-utils-lib packages installed.

These packages are necessary for this box to work with a VagrantFile configured to share folders using NFS instead of VBoxGuestAdditions

**Comment by shanethehat** [comment]
Would it be preferable to have nfs in available in the existing images or some duplicates with nfs support?

**Comment by drock** [comment]
I, personally, would prefer that they were just included in the existing images.  Vagrant's documentation and configuration positions using NFS for file sharing as a pretty much turn-key option, just set a flag to true.  Would be nice for that to be the case with the boxes as well.  

Additionally, at least with php stacks, I have found that nfs performs many times better than serving the php files from a VBox additions shared folder.  Thus, I almost exclusively use nfs for sharing the folders.

**Comment by shanethehat** [comment]
I agree with all of that, but adding nfs to all of these boxes is going to be a bit of an investment of time. It would be nice to get an opinion from @kbarber before embarking on that journey.

**Comment by mitom** [comment]
Is there any update on this?

**Comment by salimane** [comment]
+1

**Comment by salimane** [comment]
@kbarber what do you think this ?
